packages/url/src/is-file-url.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Determines whether the given string looks like a file URL.
+ *
+ * @param {string} url The string to scrutinise.
+ *
+ * @example
+ * ```js
+ * const isFileURL = isFileURL( 'file:///file.txt' ); // true
+ * ```
+ *
+ * @see https://url.spec.whatwg.org/
+ * @see https://url.spec.whatwg.org/#valid-url-string
+ *
+ * @return {boolean} Whether or not it looks like a file URL.
+ */
+export function isFileURL( url ) {
+	return url.startsWith( 'file:' );
```

Comment:
I'm still testing, but I wonder if we will have a case where `url` is undefined. In some places we check the url exists before calling this but in some other [places](https://github.com/WordPress/gutenberg/pull/20985/files#diff-f962dfb7b68050764934f0b1cc28d302R83) there aren't any checks. What do you think if we do:

```suggestion
	return url && url.startsWith( 'file:' );
```

Just to be safe =D

Reply:
Good catch, I updated the code with that suggestion.